app/scripts/modules/core/src/managed/EnvironmentsHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Dropdown } from 'react-bootstrap';
+
+import { Application } from '../application';
+import { Illustration, IllustrationName } from '../presentation';
+import { showToggleManagedResourceModal } from './ToggleManagedResourceForApplication';
+
+import './EnvironmentsHeader.less';
+
+interface IEnvironmentsHeaderProps {
+  app: Application;
+  clusterInfo: { managed: number; total: number };
+}
+
+export const ToggleManagedResourceAction = ({ app }: { app: Application }) => (
+  <li>
+    <a
+      onClick={() => {
+        showToggleManagedResourceModal({ application: app });
+      }}
+    >
+      {app.isManagementPaused ? 'Resume management' : 'Disable management'}
+    </a>
+  </li>
+);
+
+const EnvironmentHeaderInfo: {
```

Comment:
IMO going the config route here was a good decision, much simpler to mentally process. 🔍 🔍 🔍 SUPER nit: consider changing the casing of this to `environmentHeaderInfo` since it's not a component